README.md

```diff
@@ -56,6 +56,22 @@ doc = DocumentFile.from_pdf("path/to/your/doc.pdf").as_images()
 result = model(doc)
 ```
 
+### Deal with the rotation issues
+You can provide rotated documents, and/or documents with different boxes orientations.
+We provide different options to deal with such cases:
+
+- If you will only use straight document pages with straight words (horizontal, same reading direction),
+pass `assume_straigth_pages=True` in the `ocr_predictor`, it will directly fit straight boxes to your pages
+and return straight boxes, it is the faster pipeline.
+
+- If you want the predictor to output straight boxes (no matter the orientation of your pages, it will approximate
+rotated boxes to straight ones), you need to pass `export_as_straight_boxes=True` in the predictor. Otherwise, it
```

Comment:
"it will approximate rotated boxes to straight ones" --> "the final localizations will be converted to straight boxes"